test/popup.spec.js

```diff
@@ -304,6 +304,43 @@ describe('Page.Events.Popup', function() {
     expect(await popup.evaluate(() => !!window.opener)).toBe(true);
     await context.close();
   });
+  it.fail(true)('should work with Shift-clicking', async({browser, server}) => {
+    // Chromium:
+    // - Shift+Click fires frameRequestedNavigation that never materializes
+    //   because it actually opens a new window.
+    // - New window does not report an opener.
+    // WebKit: Shift+Click does not open a new window.
```

Comment:
I feel like it may be platform specific, let me check on linux.

Reply:
It's loading new page in the same window on GTK :/

Reply:
Same for me on mac.